highfidelity/hifi
# Fixed Blocks downloads (now Google Poly)

The problem: 
Since Google made Blocks a part of Poly, the URL references for download filters had to be updated.

Test plan:
1. Get the Blocks app from marketplace
2. Open Blocks
3. Select a model and click download, then select "OBJ File" and select "Download" from the pop-up
4. The model should appear in front of you
